app/views/flood_risk_engine/enrollments/_back_link.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= link_to(
       t("global.back"),
       enrollment_step_path(form.enrollment, form.enrollment.previous_step),
-      class: "back-link"
+      class: "link-back"
```

Comment:
The `back-link` class is used to identify the back link and replace it's functionality with JavaScript history step back. If you want to change the name, you also need to change it in app/assets/javascripts/flood_risk_engine/application.js